cmd/crio/config.go

```diff
@@ -89,7 +89,15 @@ runtime = "{{ .Runtime }}"
 # Path to OCI compatible runtime used for untrusted container workloads. This
 # is an optional setting, except if default_container_trust is set to
 # "untrusted".
-runtime_untrusted_workload = "{{ .RuntimeUntrustedWorkload }}"
+# DEPRECATED: use "crio.runtime.runtimes" instead. If provided, this
```

Comment:
we should deprecate `Runtime` as well and tell people to use `Runtimes` instead https://github.com/kubernetes-sigs/cri-o/pull/1822/files#diff-c9aa780302e81f9f9029ef37a74bd455R87
Also, we must blank out Runtime and use Runtimes 

I guess I'll make a follow up for that

Reply:
Done now with my latest commit